### PR TITLE
DM-3620 - lwm2m bugfix changelog y2024

### PR DESCRIPTION
**Pull request by jakubsobolewskisag**
y2024 changelog file

### CODE REVIEW

File: content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3620
+version: 10.18.540.115
+---
+When a LwM2M 1.1 Send request that included multiple timestamps is received, the LWM2M agent in some cases processed object resource mappings actions multiple times. This issue is now resolved to make mapping actions triggered only once.
```
Comment (aaronraab) [inline content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md:17]: -> "When a LwM2M 1.1 send request that includes multiple timestamps has been received, the object resource mapping actions have in some cases been processed multiple times by the LwM2M agent. This issue has been resolved to ensure that mapping actions get only triggerd once. " (Sorry, I currently don't have the option to directly commit suggestions

**Review by aaronraab — CHANGES_REQUESTED** [review]

**Review by Oezge-Akin — APPROVED** [review]

File: content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date:
+title: LWM2M agent executes object mappings multiple times on LWM2M 1.1 SEND
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md:3]:
```suggestion
title: Object mapping actions only executed once on LWM2M 1.1 send request
```

Please positively phrase in the title what works now (opposed to focusing on what was the issue).

File: content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3620
+version: 10.18.540.115
+---
+When a LwM2M 1.1 send request that includes multiple timestamps has been received, the object resource mapping actions have in some cases been processed multiple times by the LwM2M agent. This issue has been resolved to ensure that mapping actions get only triggered once.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md:17]:
```suggestion
When a LWM2M 1.1 send request that includes multiple timestamps has been received, the object resource mapping actions have in some cases been processed multiple times by the LwM2M agent. This issue has been resolved to ensure that mapping actions get only triggered once.
```

File: content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3620
+version: 10.18.540.115
+---
+When a LWM2M 1.1 send request that includes multiple timestamps has been received, the object resource mapping actions have in some cases been processed multiple times by the LwM2M agent. This issue has been resolved to ensure that mapping actions get only triggered once.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md:17]:
```suggestion
When a LWM2M 1.1 send request that includes multiple timestamps has been received, the object resource mapping actions have in some cases been processed multiple times by the LWM2M agent. This issue has been resolved to ensure that mapping actions get only triggered once.
```

File: content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date:
+title: Object mapping actions only executed once on LWM2M 1.1 send request
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity_10-18-540-115_lwm2m-agent-executes-object-mappings-multiple-times.md:3]:
```suggestion
title: Object mapping actions no longer executed multiple times on LWM2M 1.1 send request
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**